### PR TITLE
feat: add information about good first issues

### LINKED ISSUE #2527: [Edit] Subject: GitHub
### Reason/inspiration (optional)

I didn’t see Good First Issues in the GitHub chapter, so I’d like to add it. I think readers would benefit from it because they often ask how to find issues in projects.

### Entry Type (select all that apply)

- Add new content (definitions, codeblocks, etc.)
### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by CBID2**
### Description

- I added information about good first issues to the [GitHub Entry](https://www.codecademy.com/resources/docs/general/github). 
- Closes #2527 
### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/github/github.md
Comment (SSwiniarski) [inline content/general/concepts/github/github.md:23]:
```suggestion
["Good First Issue"](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) is a label open source maintainers used on GitHub to categorize issues that are intended for people who have either never made an open source contribution before or are new to their project. It also signifies to maintainers that extra assistance is needed when reviewing a new contributor's pull request, whether it's introducing the contributing guidelines or providing in-line comments on lines of code that need revision. "Good First Issue" is also a list of repositories that contain beginner-level issues in the [topics section of GitHub](https://github.com/topics/good-first-issue).
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@CBID2 I did a first review and left a line-editing eomment.

**Comment by CBID2** [comment]
> @CBID2 I did a first review and left a line-editing eomment.

Changes made @SSwiniarski! :)

**Review by SSwiniarski — APPROVED** [review]
@CBID2 I did one more slight edit and it looks good for a second review.

**Comment by CBID2** [comment]
> @CBID2 I did one more slight edit and it looks good for a second review.

Great @SSwiniarski. Can you review the one I did on URL Slugs?

**Review by KTom101 — CHANGES_REQUESTED** [review]
@CBID2 Added feedback.

**Comment by CBID2** [comment]
> @CBID2 Added feedback.

Done @KTom101.

**Review by KTom101 — APPROVED** [review]
@CBID2 Thank you 👍

**Comment by KTom101** [comment]
@CBID2 Changes are live: https://www.codecademy.com/resources/docs/general/github 🎉

cc @yangc95

**Comment by CBID2** [comment]
> @CBID2 Changes are live: https://www.codecademy.com/resources/docs/general/github 🎉
> 
> 
> 
> cc @yangc95 

Thanks @KTom101! :) Now I just need an approval rating on PR #2720

**Comment by KTom101** [comment]
@CBID2 Looks like @yangc95 is already on it 🔥 Thanks for your patience on these approvals!

**Comment by CBID2** [comment]
> @CBID2 Looks like @yangc95 is already on it 🔥 Thanks for your patience on these approvals!

You're welcome @KTom101. I was hoping you'd merge it though

**Comment by KTom101** [comment]
@CBID2 Sorry! @yangc95 may have changes already pending on their end. Please give it some more time 🙂

**Comment by CBID2** [comment]
> @CBID2 Sorry! @yangc95 may have changes already pending on their end. Please give it some more time 🙂

Ohh ok

**Pull request merged**
